fmt/testdata/ImportDirective/bracket-spacing.fmt.sol

```diff
@@ -1,4 +1,4 @@
-// config: bracket-spacing=true
+// config: bracket_spacing = true
```

Comment:
this is pretty neat, but I wonder if we should introduce a branded identifier here, so it's obvious what this is for, like `forgefmt` for example

Reply:
the logic for this is just in the tests. I think when we go around and start implementing configuration options / disabling in the files I think we can do we'll have to come up with a good system.